### DISCUSSION TITLE
Direct discussion link to load comment thread if needed

### DISCUSSION THREAD

**Discussion by IvanRibakov** [community/community#10367] [Pull Requests]
Currently, when sharing or opening from the email notification links to PR conversations  (with `#discussion_<ID>` anchor), said link doesn't do anything if:
1. comment has been marked as resolved
2. comment is hidden behind "load more" panel

This really breaks the user experience and makes keeping track of multiple parallel conversations (the norm for PR review process) that much more difficult - not only all comments must be manually loaded, but they also have to be scrolled through and manually checked in order to find the necessary conversation thread.

Discussion permalink should always result in the linked comment and associated thread to be loaded and visible (autoscrolled to if needed).

**Comment by feuGeneA** [comment]
Just now I tried to link a co-worker to a comment thread, but he couldn't follow the link because it was behind a "load more" panel. In theory he could expand all of the "load more" panels (of which there are many in this PR), and then go to that link again... but what a pain! I ended up having to send him a screenshot of the conversation instead.

This is a real usability _problem_, that's affecting real users, today. Hyperlinks should be addressable. Please fix this.

(I'm new to GitHub Discussions. Am I abusing it? I came here to "discuss", but now my comment is being treated as an "Answer"? Confusing. Apologies in advance if I'm writing out of context here.)

**Reply by IvanRibakov** [reply]
> In theory he could expand all of the "load more" panels (of which there are many in this PR), and then go to that link again

Not sure that would work either. So far I haven't found a way to trigger the "go to link" action manually. It seems to be triggered automatically on page load **IF** the anchor the link is pointing to is in the (visible?) DOM. So after you've "loaded more", the only way to go to the link is to reload the page...oh wait, that takes you back to square one...

If someone knows how to trigger the "go to link" action without page reload - I'm all ears!

**Reply by IvanRibakov** [reply]
> Am I abusing it?

Don't think so, I've seen similar usage pattern in other discussion topics.

> I'm new to GitHub Discussions.

If you haven't already, please consider upvoting this discussion (up arrow with counter under the original post) - helps raise the visibility of this problem.

**Reply by feuGeneA** [reply]
I already upvoted, and got my co-workers to upvote as well. :)

Here's what works successfully for me:

1. Click the link that doesn't work, which takes you to the top-level PR view.
2. expand all of the "load more" panels
3. don't refresh, but rather: click in the address bar, where the full thread hyperlink is still displayed, and just press enter

I've been using this procedure regularly, on Firefox 78.13. But I just tried with Chrome and it does NOT work there :sob:

**Reply by BlindChickens** [reply]
Is there anyone with new insights into this problem?

I don't see the nuance in this? Are there not a thousand reasons to have a permalink to a discussion just work, no matter how old, or resolved?

Please tell me there is another more active discussion or issue regarding this?
